src/ucp/core/ucp_rkey.c

```diff
@@ -667,7 +715,9 @@ ucp_memh_pack_internal(ucp_mem_h memh, const ucp_memh_pack_params_t *params,
 
     UCP_THREAD_CS_ENTER(&context->mt_lock);
 
-    size = ucp_memh_packed_size(memh, flags, rkey_compat);
+    sys_dev_map = ucp_memh_sys_dev_map(memh);
```

Comment:
nextpr: why do we need sys_dev_map, can't we just understand from memh->sys_dev if need to pack lane distances or not?